passes/cmds/setundef.cc

```diff
@@ -188,34 +262,100 @@ struct SetundefPass : public Pass {
 				if (!module->processes.empty())
 					log_error("The 'setundef' command can't operate in -undriven mode on modules with processes. Run 'proc' first.\n");
 
-				SigMap sigmap(module);
-				SigPool undriven_signals;
-
-				for (auto &it : module->wires_)
-					undriven_signals.add(sigmap(it.second));
-
-				for (auto &it : module->wires_)
-					if (it.second->port_input)
-						undriven_signals.del(sigmap(it.second));
-
-				CellTypes ct(design);
-				for (auto &it : module->cells_)
-				for (auto &conn : it.second->connections())
-					if (!ct.cell_known(it.second->type) || ct.cell_output(it.second->type, conn.first))
-						undriven_signals.del(sigmap(conn.second));
-
-				RTLIL::SigSpec sig = undriven_signals.export_all();
-				for (auto &c : sig.chunks()) {
-					RTLIL::SigSpec bits;
-					if (worker.next_bit_mode == MODE_ANYSEQ)
-						bits = module->Anyseq(NEW_ID, c.width);
-					else if (worker.next_bit_mode == MODE_ANYCONST)
-						bits = module->Anyconst(NEW_ID, c.width);
-					else
-						for (int i = 0; i < c.width; i++)
-							bits.append(worker.next_bit());
-					module->connect(RTLIL::SigSig(c, bits));
-				}
+        if (expose_mode) {
```

Comment:
Please fix the indenting. See https://github.com/YosysHQ/yosys/blob/master/CodingReadme#L218